.gitignore

```diff
@@ -18,3 +18,6 @@ coverage
 
 # system-related
 .DS_Store
+
+# wundergraph
+wundergraph/.wundergraph/generated
```

Comment:
It is advised to create its own `.gitignore` per service, so they can be easily renamed or copied over